simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+
+import launch
+import launch_ros.actions
+from ament_index_python.packages import get_package_share_directory
+
+
+def generate_launch_description():
+    ld = launch.LaunchDescription([
+        launch.actions.DeclareLaunchArgument(
+            name='model',
+            default_value=launch.substitutions.EnvironmentVariable(
+                'TURTLEBOT3_MODEL'),
+            description='model type [burger, waffle, waffle_pi]'
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='map_file',
+            default_value=os.path.join(get_package_share_directory(
+                'turtlebot3_navigation') + 'maps', 'map.yaml')
+        ),
+        launch.actions.DeclareLaunchArgument(
+            name='open_rviz',
+            default_value='true'
```

Comment:
I feel like the default to this should be false since we set it to that anyway.

Reply:
This was taken from https://github.com/aws-robotics/aws-robomaker-small-house-world/blob/master/launch/turtlebot3_navigation.launch where it was set to true previously. Wanted to keep it consistent with ROS1.